docs/source/configuring.rst

```diff
@@ -22,10 +30,28 @@ When deploying, the `scw-serverless` CLI will look for a Serverless instance in
 Functions
 ---------
 
-To configure your serverless functions, you can provide keyword arguments to the decorators.
+To configure your serverless functions, you can provide keyword arguments to the decorators. The Function name that will appear in Scaleway console will be the name of your the function's handler.
 
 .. autoclass:: scw_serverless.config.function.FunctionKwargs
 
+.. code-block:: python
+
+   app = Serverless("example")
+   app.func(
+      privacy="public",
+      env={"key": "value"},
+      secret={"key": "value"},
+      min_scale= 0,
+      max_scale= 5,
+      memory_limit= 256,
+      timeout= 300,
+      description= "Lores Ipsum",
+      http_option= "enabled" #https only
+   )
+   def handler(event, context):
+      # Do Things
+      return {"message": "Hello World"}
+
 Triggers
 --------
 
```

Comment:
On Cron trigger:
- Is the function private by default ? 
- What is the difference between cron and the schedule() Method in the App object ?
- Add an example

Reply:
- No, I think it will be made public currently if you don't specify. That doesn't make a lot of sense indeed for crons. I will change it so that `@app.schedule()` returns sets the privacy to private if not specified.
- The CronTrigger class can be passed to the `@app.schedule()` instead of a string. The only benefit of that is to pass a name in addition to the cron schedule.

I don't know why the CronTrigger section only talks about this class because ultimately we want people to use `@app.schedule("0 1 * * * *")` to make their crons and not `@app.func(triggers=[CronTrigger(schedule="0 1 * * * *")], name="toto")` I will update the docs to reflect this.

I really the idea of adding the examples directly into the docs as you did, it's really nice :100: